src/Raven.Server/Commercial/LetsEncrypt/CompleteAuthorizationAndGetCertificateParameters.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Threading;
+
+namespace Raven.Server.Commercial.LetsEncrypt;
```

Comment:
Is it correct namespece? Is this class specific to Let's encrypt? If not then maybe `Raven.Server.Commercial.SetupWizadr` would be better location for it

Reply:
done